CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changes
 
 * [#6688](https://github.com/rubocop-hq/rubocop/pull/6688): Add `iterator?` to deprecated methods and prefer `block_given?` instead. ([@tejasbubane][])
+* [#6791](https://github.com/rubocop-hq/rubocop/pull/6791): Allow `Rails/ReflectionClassName` to use symbol argument. ([@unasuke][])
```

Comment:
Ah, probably this change is a bug fix. The symbol object is a valid argument that will not cause the following error.
https://github.com/rubocop-hq/rubocop/pull/6704#issuecomment-457121739

Reply:
Wow :exclamation: I moved the line to `fix bug` section, thx :wink: